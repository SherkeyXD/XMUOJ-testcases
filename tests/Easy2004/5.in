50
137
-18
-50
60
94
17
30
102
29
45
-50
-25
20
-6
40
2
120
112
59
84
6
-42
-4
94
142
109
37
100
93
-13
74
145
108
-36
135
120
-34
-8
44
90
133
136
-8
130
17
19
105
48
114
8
114
85
0
60
94
140
-22
50
90
106
50
66
122
94
26
76
120
144
2
48
-28
116
31
-14
34
70
-6
-30
118
50
22
119
113
40
123
25
99
64
-45
50
-47