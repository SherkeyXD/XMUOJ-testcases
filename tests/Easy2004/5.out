E
Score is error!
Score is error!
Score is error!
D
A
E
E
Score is error!
E
E
Score is error!
Score is error!
E
Score is error!
E
E
Score is error!
Score is error!
E
B
E
Score is error!
Score is error!
A
Score is error!
Score is error!
E
A
A
Score is error!
C
Score is error!
Score is error!
Score is error!
Score is error!
Score is error!
Score is error!
Score is error!
E
A
Score is error!
Score is error!
Score is error!
Score is error!
E
E
Score is error!
E
Score is error!
E
Score is error!
B
E
D
A
Score is error!
Score is error!
E
A
Score is error!
E
D
Score is error!
A
E
C
Score is error!
Score is error!
E
E
Score is error!
Score is error!
E
Score is error!
E
C
Score is error!
Score is error!
Score is error!
E
E
Score is error!
Score is error!
E
Score is error!
E
A
D
Score is error!
E
Score is error!
